src/hiscores.ts

```diff
@@ -97,11 +97,16 @@ export function parseStats(csv: string): Stats {
       return activity;
     });
 
+  const filteredBosses =
+    mode === 'seasonal'
+      ? BOSSES.filter((boss) => boss !== 'tempoross')
+      : BOSSES;
```

Comment:
The main fix. Kind of annoying it has to handle the one edge case here but it is what it is :/

Reply:
Yeah that is a strange edge case. Can you add a comment above `filteredBosses` in order to explain the behavior? Plus it will flag it for future removal once they fix this on their side.

Reply:
After verifying this edge case myself, I believe the missing boss is actually the newly added `theatreOfBloodHardMode`. If you compare the stats of `Fysad` in seasonal (on the left) to the normal game:

![image](https://user-images.githubusercontent.com/13290954/121026617-4bfdf380-c774-11eb-84de-49ada12c889c.png)

Line 70 on both sides is The Gauntlet, while Zulrah is the last line for both sides. Since The Gauntlet is below Tempoross on the list, it has to be another boss which is omitted.
Then I visited the hiscores page, and clicked on "Leagues" and sure enough TOB: Hard Mode was missing from the side bar:
https://secure.runescape.com/m=hiscore_oldschool_seasonal/overall

![image](https://user-images.githubusercontent.com/13290954/121027459-f9710700-c774-11eb-95b3-9ff73e4044de.png)

Reply:
Oh shoot you're right 🤦‍♂️ I'll update this after work today.